containers/docker-compose.yaml

```diff
@@ -5,19 +5,23 @@ services:
     restart: always
     ports:
       - 5430:5432
-    volumes:
-      - db_data:/var/lib/postgresql/data
```

Comment:
I removed the volumes here and because I realised [data is persisted across runs](https://github.com/docker/compose/issues/2127) in a way that was surprising to me. I checked that the psql and mssql benchmarks still run ok, but could this have any implications that I'm not considering?

Reply:
I don't blame you for this -- it's typical in Docker Compose setups you want volumes persisted across runs for databases, otherwise you lose your DB every time the stack is reset.

I didn't think to mention to you, you need to run `docker-compose down -v` to stop all contains AND remove the volumes.

A "true" up/down cycle for Docker Compose is:
`docker-compose up`
`docker-compose down -v --remove-orphans`

```ps1
Options:
[...]
    -v, --volumes           Remove named volumes declared in the `volumes`
                            section of the Compose file and anonymous volumes
                            attached to containers.
    --remove-orphans        Remove containers for services not defined in the
                            Compose file
```

I can see how persistent volumes and data would be undesirable in a benchmarking scenario.
So I am inclined to agree with this change, and to opt to let users to mount the volume if they want, good call 👍